src/components/sales/OrderBill.vue

```diff
@@ -15,8 +15,8 @@
 		</div>
 
 		<div id="date-section" class="mb-5">
-			<p class="text-center">3:44 pm</p>
-			<p class="text-center">Friday, 11 June, 2021</p>
+			<p class="text-center">{{getCurrentTime()}}</p>
+			<p class="text-center">{{getCurrentDate()}}</p>
```

Comment:
Use Order field createdOn. Do not use current date.

Reply:
I can do that but shouldn't the printing date/time be on the bill.

Reply:
change done.